src/main/java/com/github/maxopoly/WurstCivTools/ConfigParser.java

```diff
@@ -57,6 +54,10 @@ public void parseTools(ConfigurationSection config) {
 			WurstEffect effect;
 			switch (type) {
 			case "PYLONFINDER":
+				if (current.getBoolean("enabled",false) == false){
```

Comment:
This isn't needed, because you can just chose not to specify any pylon finders. The list of tools may also have different config variations of the same effect, so this doesn't really make sense.